_ml-commons-plugin/connectors.md

```diff
@@ -0,0 +1,392 @@
+
+- [Amazon SageMaker](https://aws.amazon.com/sagemaker/) allows you to host and manage the life cycle of text embedding models, powering semantic search queries in OpenSearch. When connected, Amazon SageMaker hosts your models and OpenSearch is used to query inferences. This benefits Amazon SageMaker users who value its functionality such as model monitoring, serverless hosting, and workflow automation for continuous training and deployment.
+- [Amazon Comprehend](https://aws.amazon.com/comprehend/) allows you to extract metadata automatically from an OpenSearch ingest pipeline, enriching data indexed in OpenSearch and improving your search capabilities. 
+- [ChatGPT](https://openai.com/blog/chatgpt) enables you to run OpenSearch queries while invoking the ChatGPT API, helping you build on OpenSearch faster and improves the data retrieval speed for OpenSearch search functionality.
+- [Amazon Bedrock](https://aws.amazon.com/bedrock/) is an alternative to ChatGPT inside a fully managed service.
+- [NVIDIA Triton](https://developer.nvidia.com/triton-inference-server) allows you to host text embedding models on NVIDIA's high performance model-serving technology. Power semantic search queries and vector generation ingest pipelines using [GPU-acceleration]({{site.url}}{{site.baseurl}}/ml-commons-plugin/gpu-acceleration/).
+- [Cohere](https://cohere.com/) allows you to use data from OpenSearch to power Cohere's large language models.
+
+## Prerequisites
+
+If you are an admin deploying an ML connector, make sure that target model of the connector has already been deployed on your chosen platform. Furthermore, make sure that you have permissions to send and receive data to the third-party API for your connector. 
+
+When access control is enabled on your third-party platform, you can enter your security settings using the `authorization` or `credential` settings inside the connector API.
+
+### Adding trusted endpoints
+
+To configure connectors in OpenSearch, add the trusted endpoints into your cluster settings using the `plugins.ml_commons.trusted_connector_endpoints_regex` setting, as shown in the following example:
+
+```json
+PUT /_cluster/settings
+{
+    "persistent": {
+        "plugins.ml_commons.trusted_connector_endpoints_regex": [
+            "^https://runtime\\.sagemaker\\..*\\.amazonaws\\.com/.*$",
+            "^https://api\\.openai\\.com/.*$",
+            "^https://api\\.cohere\\.ai/.*$",
+            "^https://bedrock\\..*\\.amazonaws.com/.*$"
+        ]
+    }
+}
+```
+{% include copy-curl.html %}
+
+### Enabling ML nodes
+
+Connectors require the use of dedicated ML nodes. To make sure you have ML nodes enabled, update the following cluster settings:
+
+```json
+PUT /_cluster/settings
+{
+    "persistent": {
+        "plugins.ml_commons.only_run_on_ml_node": true,
+        "plugins.ml_commons.native_memory_threshold": 100
```

Comment:
Native memory threshold doesn't have impact on remote inference, it's not a necessary to explicitly specify this, and setting this value to 100 can affect circuit breaker logic for local model, it's not a recommendation to  set this to 100.

Reply:
I'll go ahead and remove this line from the example.